logic.js

```diff
@@ -0,0 +1,58 @@
+const Tweeter = function () {
+    const posts = [
+        {
+            text: "First post!",
+            id: "p1",
+            comments: [
+                { id: "c1", text: "First comment on first post!" },
+                { id: "c2", text: "Second comment on first post!!" },
+                { id: "c3", text: "Third comment on first post!!!" }
+            ]
+        },
+        {
+            text: "Aw man, I wanted to be first",
+            id: "p2",
+            comments: [
+                { id: "c4", text: "Don't wory second poster, you'll be first one day." },
+                { id: "c5", text: "Yeah, believe in yourself!" },
+                { id: "c6", text: "Haha second place what a joke." }
+            ]
+        }
+    ]
+
+    let postIdCounter = 2
+    let commentIdCounter = 6
+
+    const getPosts = () => posts
+
+    const addPost = (content) => posts.push({ text: content, id: "p" + (++postIdCounter), comments: [] })
+
+    const removePost = function (pid) {
```

Comment:
even though using `pid` is shorter, it is better to name variables what they actually are. You want people to be able to look at your code and easily read it. 

instead, do `postId` it is a bit longer but helps make things more clean and readable